src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -270,8 +268,8 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, EvalVisitorCo
                 result = await customTexlFunc.InvokeAsync(FunctionServices, args, _cancellationToken).ConfigureAwait(false);
             }
             else
-            {               
-                if (FunctionImplementations.TryGetValue(func, out var ptr))
+            {
+                if (FunctionImplementations.TryGetValue(func, out AsyncFunctionPtr ptr))

```

Comment:
>AsyncFunctionPtr [](http://example.com/codeflow?start=66&length=16)

Why change from `var`?

Reply:
It clearly shows the type :)
Here AsyncFunctionPtr, while 10 lines above it's IAsyncTexlFunction.
I believe it makes the code more readable (at least it helps me)